### DISCUSSION TITLE
loading json file

### DISCUSSION THREAD

**Discussion by ashokchokalingam** [Yamato-Security/hayabusa#1483] [General]
Hi,

I ma having an opensource log management which will send the logs over syslog and i convert it using logstash to json format and i would like to look for sigma rules using hayabusa. how can i load json log file as an input. The below is the output i get.
![image](https://github.com/user-attachments/assets/83adfbec-f1dc-40f0-922e-7bfe0410ed1e)

**Comment by YamatoSecurity** [comment]
I don't think it can handle those kind of custom JSON files. Only JSON exported from HELK or Splunk at the moment.

**Comment by ashokchokalingam** [comment]
Thank you so much for the reply, how do i load splunk or elk exported data?
could you share the syntax if possible?


Regards
Ashok
